scripts/actions/__tests__/add-files-to-translation-queue.test.js

```diff
@@ -0,0 +1,256 @@
+import fs from 'fs';
+import AWS from 'aws-sdk';
+import fetch from 'node-fetch';
```

Comment:
I think we'll need to change these imports over to the old `require` syntax, since this will be running on node 12 in the Github runners.